plugins/ltac/g_obligations.mlg

```diff
@@ -10,7 +10,7 @@
 
 (*
   Syntax for the subtac terms and types.
-  Elaborated from correctness/psyntax.ml4 by Jean-Christophe Filliâtre *)
+  Elaborated from correctness/psyntax.mlg by Jean-Christophe Filliâtre *)
```

Comment:
Oops, I missed this one before.  I suspect this one is also historical, and thus better left alone.

Reply:
No problem :) I'm also reverting this one